src/pages/CriticalIssueEntry.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  // @ts-ignore no declarations
+  import { initAll } from "govuk-frontend";
+  import "../style/main.css";
+  import type { LngLat } from "maplibre-gl";
+  import { onMount } from "svelte";
+  import { Layout, MapLibreMap, StreetViewController } from "../lib/common";
+  import BaselayerSwitcher from "../lib/critical_entry/BaselayerSwitcher.svelte";
+  import Form from "../lib/critical_entry/Form.svelte";
+  import Pin from "../lib/critical_entry/Pin.svelte";
+
+  onMount(() => {
+    // For govuk components. Must happen here.
+    initAll();
+  });
+
+  let markerPosition: LngLat | null = null;
+  let streetviewOff = true;
```

Comment:
Bit of odd mutual exclusion logic here. If stview is on, clicking shouldn't create a new pin. And while stview is on, you can't switch basemap, because the road helper stuff would then need to detect that change and fiddle with the correct layers.